src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -79,12 +85,15 @@ def test_WINNF_FT_S_SSS_8(self, config_filename):
     Checks that SAS UUT response with fatal alert message.
     """
     config = loadConfig(config_filename)
+    certificate_hash = getCertificateFingerprint(config['sasCert'])
+    self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,
+                                   'url': SAS_TH_URL })
     self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
                                    client_key=config['sasKey'])
 
   def generate_SSS_9_default_config(self, filename):
     """Generates the WinnForum configuration for SSS_9"""
-    # Create the actual config for domain proxy cert/key path
+    # Create the actual config for sas cert/key path
```

Comment:
"SAS" (all caps) here and below.

Reply:
All the   occurrences  of the "sas" in   the comments are made uppercase "SAS"